### DISCUSSION TITLE
A custom tag for a collection of projects

### DISCUSSION THREAD

**Discussion by robertsawko** [GothenburgBitFactory/taskwarrior#3202] [Q&A]
How to configure Taskwarrior to apply a certain tag to every task of a given set of projects? My use case is an ability to differentiate between work projects and home projects.

**Comment by smemsh** [comment]
- contexts may satisfy your need
- you might just add +home or +work when you do the add
- you can encode home/work into project hierarchy ie home.proj1 or work.proj1
- you could use an add hook to check against some hardcoded list of projects and add a tag
